packages/account-v2/src/utils/pooUtils.ts

```diff
@@ -30,3 +38,26 @@ export const generatePOOInitialValues = (paymentMethodData: TPaymentMethodData)
         return acc;
     }, {} as TProofOfOwnershipFormValue);
 };
+
+const maskCardNumber = (cardNumber: string) => {
+    if (
```

Comment:
can we move this condition out from if statement ?

Reply:
Are u referring to making it ternary operator? The condition is huge, so I kept it in IF block